packages/core/src/textInput/private/Input.tsx

```diff
@@ -88,23 +90,26 @@ const Input = styled.input.attrs(({ type }) => ({
       },
     },
     inputSize === 'medium' && {
-      padding: `${rem(23)} ${rem(15)} ${rem(7)}`,
+      padding: `${isLabelPresent ? rem(23) : rem(15)} ${rem(15)} ${
+        isLabelPresent ? rem(7) : rem(15)
+      }`,
       paddingRight: type === 'password' ? rem(55) : rem(15),
-      '&:not(:focus)::placeholder': {
-        opacity: 0,
-      },
       '&:not(:focus):placeholder-shown + label': {
         top: '50%',
         marginTop: rem(-7),
         fontSize: rem(14),
         lineHeight: rem(14),
       },
       '&:hover:not(:focus):not([disabled])': {
-        padding: `${rem(22)} ${rem(14)} ${rem(6)}`,
+        padding: `${isLabelPresent ? rem(22) : rem(14)} ${rem(14)} ${
```

Comment:
Can you write it more readable way like:
```
padding: ${isLabelPresent} ? `rem(22) rem(6)` : `rem(14`
```

Reply:
This was formatted by the auto formatter. Maybe I will create a separate function instead.

Reply:
imo smth like this would be easy to read:
`padding: isLabelPresent ? "rem(X) ... ..." : "rem(Z) ... ..." `